main.go

```diff
@@ -12,13 +12,14 @@ import (
 )
 
 var (
-	version    = "dev"
-	app        = kingpin.New("svu", "semantic version util")
-	nextCmd    = app.Command("next", "prints the next version based on the git log").Alias("n").Default()
-	majorCmd   = app.Command("major", "new major version")
-	minorCmd   = app.Command("minor", "new minor version").Alias("m")
-	patchCmd   = app.Command("patch", "new patch version").Alias("p")
-	currentCmd = app.Command("current", "prints current version").Alias("c")
+	version         = "dev"
+	app             = kingpin.New("svu", "semantic version util")
+	nextCmd         = app.Command("next", "prints the next version based on the git log").Alias("n").Default()
+	majorCmd        = app.Command("major", "new major version")
+	minorCmd        = app.Command("minor", "new minor version").Alias("m")
+	patchCmd        = app.Command("patch", "new patch version").Alias("p")
+	currentCmd      = app.Command("current", "prints current version").Alias("c")
+	onlyCoreVersion = app.Flag("only-core-version", "discards pre-release and build metadata").Bool()
```

Comment:
this is a bit confusing, maybe a `--metadata` flag, which defaults to `true`, then the user can use `--no-metadata`? 🤔

Reply:
I was sticking to the terminology used by in the semver spec itself:

<img width="993" alt="Screenshot 2020-11-10 at 13 34 42" src="https://user-images.githubusercontent.com/582281/98674769-942d7280-2359-11eb-888c-b14d4d2db508.png">

We could even have one flag for build metadata and one for the pre-release as well:

`v1.0.0-alpha+build.f902daf` + `--no-metadata ` -> `v1.0.0`
`v1.0.0-alpha+build.f902daf` + `--no-build` -> `v1.0.0-alpha`
`v1.0.0-alpha+build.f902daf` + `--no-build --no-pre-release` -> `v1.0.0`